homeassistant/components/thermostat/radiotherm.py

```diff
@@ -0,0 +1,127 @@
+"""
+homeassistant.components.thermostat.radiotherm
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Adds support for Radio Thermostat wifi-enabled home thermostats
+"""
+import logging
+import datetime
+from urllib.error import URLError
+
+from homeassistant.components.thermostat import (ThermostatDevice, STATE_COOL,
+                                                 STATE_IDLE, STATE_HEAT)
+from homeassistant.const import (CONF_HOST, CONF_NAME, TEMP_FAHRENHEIT)
+
+REQUIREMENTS = ['radiotherm==1.2']
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """ Sets up the Radio Thermostat. """
+    logger = logging.getLogger(__name__)
+
+    try:
```

Comment:
You can remove this check, having it in `REQUIREMENTS` will ensure HA will not call this method without it being installed.